upgrade/upgrade-scripts/upgrade-container

```diff
@@ -119,6 +119,13 @@ function create_upgrade_container() {
 			"$ROOTFS_DATASET" "$ROOTFS_DATASET@$SNAPSHOT_NAME"
 		copy_optional_dataset_property "$PROP_HOTFIX_VERSION" \
 			"$ROOTFS_DATASET" "$ROOTFS_DATASET@$SNAPSHOT_NAME"
+		#
+		# The minimum version property was introduced in 6.0.11.0.
+		#
+		if compare_versions "$(get_current_version)" ge "6.0.11.0"; then
```

Comment:
I'm assuming this will be backported to 6.0/stage and not 6.0/release, so this should be 6.0.12.0 instead